packages/docusaurus-plugin-content-docs/src/__tests__/pluginOptionSchema.test.ts

```diff
@@ -9,12 +9,14 @@ import {PluginOptionSchema, DEFAULT_OPTIONS} from '../pluginOptionSchema';
 import {normalizePluginOptions} from '@docusaurus/utils-validation';
 
 // the type of remark/rehype plugins is function
-const remarkRehypePluginStub = () => {};
+const markdownPluginsFunctionStub = () => {};
+const markdownPluginsObjectStub = {};
 
 describe('normalizeDocsPluginOptions', () => {
   test('should return default options for undefined user options', async () => {
-    const {value} = await PluginOptionSchema.validate({});
+    const {value, error} = await PluginOptionSchema.validate({});
     expect(value).toEqual(DEFAULT_OPTIONS);
+    expect(error).toBe(undefined);
```

Comment:
It is possible for value to match when there is an error, so it is safer to include this